### ISSUE TITLE
Error - spotify save

### ISSUE THREAD

**Issue by daveletourneau** [ledesmablt/spotify-cli#21]
`spotify save .` returns an error for _track_ (`--track` or empty parameter) and _album_ (`--album`). It works well to save _album_ or _playlist_. Here is the error for `spotify save .` (same for `--track` and `--album`:

```
Traceback (most recent call last):
  File "/home/dave/.local/bin/spotify", line 8, in <module>
    sys.exit(cli())
  File "/usr/lib/python3/dist-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/dave/.local/lib/python3.8/site-packages/cli/commands/save.py", line 73, in save
    ', '.join(parse_artists(music['artists'])['names']), 30
KeyError: 'artists'
```
OS: Ubuntu 20.04.2
Python: 3.8.10

Thanks for this nice tool by the way!

**Comment by ledesmablt** [comment]
Fixed in [v0.3.10](https://github.com/ledesmablt/spotify-cli/releases/tag/v0.3.10). Looks like it was an API change since this was working perfectly fine before.

Please close this issue if it works on your end. Thanks!!

**Comment by daveletourneau** [comment]
Everything's ok now. Thanks!
